docs/UserGuide.md

```diff
@@ -34,12 +34,12 @@ As most applications on the market are not CLI-based, this may come across
 as new to some users. We strongly recommend that first-time users read through the user guide to gain a basic understanding
 of how to use our app. We have also included links to navigate around the user guide to allow easier access
 of related information. Some tips and tricks, as well as cautions are included throughout the guide to
-help you get more out of our application and avoid certain common dangers. Also, links back to the top of the page
-like below are present at the end of every section. To allow for easier navigation 
-around the user guide, we suggest using the table of contents as well as these links to quickly
-find and access the information needed. 
+help you get more out of our application. Additionally, links back to the top of the page
+like below are present at the end of every section that bring you back to the start of our user guide.
```

Comment:
i think there's repetition eh. "Additionally, links that bring you back to the start of our user guide are present at the end of every section." is sufficient